docs/dev/plugins.md

```diff
@@ -53,7 +90,7 @@ commandTree.listen('/sample/echo', sayHello, { docs: 'Say hello!' })
   3. Optionally, a plugin may also export a programmatic API. This API
```

Comment:
Is headless mode available automatically for a plugin?

Reply:
yes, a plugin can express that it requires electron, or can refuse to operate in a browser. without any special annotations, a plugin will behave as an ascii contributor to headless mode.

mengting, make sure to document these annotations, maybe in a future PR, so that we can get the updated sample plugin out there quickly.